demo/admin/src/generator/utils/camelCaseToHumanReadable.ts

```diff
@@ -0,0 +1,4 @@
+export function camelCaseToHumanReadable(s: string) {
+    const words = s.match(/[A-Za-z][a-z]*/g) || [];
+    return words.map((word) => word.charAt(0).toUpperCase() + word.substring(1)).join(" ");
+}
```

Comment:
Can we use [change-case](https://www.npmjs.com/package/change-case) for this? camelCase to Capital Case? We already have the dependency in the Starter (and Demo) Admin. 

Though it might cause problems because of the ESM module when moved to the library?

Reply:
[4e2fab6](https://github.com/vivid-planet/comet/pull/1599/commits/4e2fab60bd9c2d4bb6386193e81c34df475e069a)